numpy/distutils/ccompiler.py

```diff
@@ -271,12 +271,8 @@ def CCompiler_compile(self, sources, output_dir=None, macros=None,
 
     if not sources:
         return []
-    # FIXME:RELATIVE_IMPORT
```

Comment:
@mattip I'm happy to unify the relative imports either way, I assumed that absolute was preferred because of this comment here.